src/vivarium_csu_ltbi/tools/results.py

```diff
@@ -0,0 +1,117 @@
+import functools
+from pathlib import Path
+from typing import Tuple, Dict
+
+import pandas as pd
+from loguru import logger
+
+import vivarium_csu_ltbi.paths as ltbi_paths
+from vivarium_csu_ltbi import globals as project_globals
+from vivarium_csu_ltbi.results_processing import counts_output, table_output
+
+
+def validate_process_latest_results_args(model_versions: Tuple[str], location: str):
```

Comment:
I believe that should not be possible because the click decorations mandate model_versions as a required argument, but that's a fair point, it's one layer above. I'll add it